movingSquare.js

```diff
@@ -0,0 +1,93 @@
+const square = document.createElement('div')
+const h6 = document.createElement('h6')
+
+let positionX = 0
+let positionY = 0
+
+square.style.backgroundColor = 'red'
+square.style.width = '100px'
+square.style.height = '100px'
+square.style.position = 'absolute'
+square.style.left = positionX + 'px'
+square.style.top = positionY + 'px'
+square.style.zIndex = 1
+
+h6.innerHTML = 'X=' + String(positionX) + ' | ' + 'Y=' + String(positionY)
+h6.style.position = 'absolute'
+h6.style.left = 0
+h6.style.top = 0
+h6.style.margin = '4px'
+h6.style.zIndex = 2
```

Comment:
Same thing as for square. This should go to a new function `createHeadline` for example.

And I would argue that it is a headline, it is a simple text, I would go with p.